constants.js

```diff
@@ -48,17 +49,21 @@ const checksWhitelist = {
       wipCheck,
       jobCheck,
       modelCheck,
-      prTemplateCheck
+      prTemplateCheck,
+      codeOwnerCheck
     ],
     [reopenEvent]: [
       changelogCheck,
       branchCheck,
       wipCheck,
       jobCheck,
       modelCheck,
-      prTemplateCheck
+      prTemplateCheck,
+      codeOwnerCheck
     ],
     [PRLabelEvent]: [assigneeCheck, prLabelCheck],
+    [synchronizeEvent]: [mergeConflictCheck, jobCheck, modelCheck, codeOwnerCheck],
+    [closeEvent]: [allMergeConflictCheck],
     [synchronizeEvent]: [mergeConflictCheck, jobCheck, modelCheck],
```

Comment:
Why do we synchronize twice?

Reply:
My bad.
I think this error was due to a merge conflict I had.